servicex/servicex.py

```diff
@@ -483,7 +507,28 @@ async def get_data_parquet_uri_stream(
             selection_query, "parquet", title, as_signed_url
         ):  # type: ignore
             yield f_info
+            
+    async def get_data_parquet_uri_async(
+        self,
+        selection_query: str,
+        title: Optional[str] = None,
+        as_signed_url: Optional[bool] = False,
+    ) -> list:
+        """Returns a list of each of the files from the minio bucket,
+        as the files are added there.
```

Comment:
Remove the "as the files are added there" - as this is going to wait until they are done and return all of them!